book/team.yaml

```diff
@@ -443,4 +443,27 @@ people:
         link: https://github.com/zachkeskinen
     user_groups:
       - Tutorial Lead
-   
+  # =============
+  - title: Melissa Wrzesien
+    avatar: https://github.com/mlwrzesien.png
+    role: Assistant Research Scientist
+    organizations:
+      - name: NASA GSFC / University of Maryland
+    bio: >
+      Melissa's work aims to improve land surface models so they produce better
+      estimates of snow properities. Her recent work involves preparing a model
```

Comment:
@mlwrzesien it looks like the spell checker caught this incorrect spelling of "properties". Can you edit this and then I can approve? Thanks!